src/shorah/amplicon.py

```diff
@@ -53,12 +53,21 @@
 diri_exe = resource_filename(__name__, 'bin/diri_sampler')
 b2w_exe = resource_filename(__name__, 'bin/b2w')
 
+if not os.path.exists(diri_exe) or not os.path.exists(b2w_exe):
+    for path in os.environ["PATH"].split(os.pathsep):
```

Comment:
Code seems correct and does what you want it to do, but I would try to use some standard library stuff instead manually searching it with handcrafted code.

Reply:
I was looking for a solution and ended up here: https://stackoverflow.com/questions/377017/test-if-executable-exists-in-python.

Reply:
Apparently, we looked at the exact same StackOverflow, but decided to pick different solution. ( Handcrafted vs Standard library)